ag-grid-webcomponent/cell_editor/DecimalEditor.js

```diff
@@ -0,0 +1,59 @@
+export default class DecimalEditor {
```

Comment:
Looks to me like everything this editor currently does can also be done with a `valueSetter`. But we probably want to have more logic in here like only allowing numeric input like this one: https://www.ag-grid.com/javascript-data-grid/component-cell-editor/#simple-cell-editor-example
and formatting too. Basically bring in our numeric input component.